src/VotesPartialDelegationUpgradeable.sol

```diff
@@ -353,7 +386,11 @@ abstract contract VotesPartialDelegationUpgradeable is
     }
   }
 
-  /// @dev Assumes both _old and _new are sorted by DelegationAdjustment._delegatee
+  /**
+   * @dev Given an old delegation array and a new delegation array, determine which delegations have changed, create new
+   * voting checkpoints, and emit a {DelegateVotesChanged} event. Takes care to avoid duplicates and no-ops.
```

Comment:
what do you mean by `no-ops` here?

Reply:
no-op is when, for example, a delegatee appears with the same numerator in both the old and new partial delegation arrays